Junit/src/IPL.java

```diff
@@ -0,0 +1,364 @@
+    double strikeRate;
+    int wickets;
+
+    Player(String name,String team,String role,int matches,int runs,double average,double strikeRate,int wickets){
+        this.name = name;
+        this.team = team;
+        this.role = role;
+        this.matches = matches;
+        this.runs  =runs;
+        this.average = average;
+        this.strikeRate  = strikeRate;
+        this.wickets = wickets;
+    }
+
+
+
+    public String getName(){
+        return name;
+    }
+
+    public String getRole(){
+        return role;
+    }
+
+    public int getWicket(){
+        return wickets;
+    }
+
+    public String getTeam(){
+        return team;
+    }
+
+    public int getRuns(){
+        return runs;
+    }
+
+    public int compareTo(Player p){
+        return 0;
+    }
+
+    // for all rounders
+    public int cumulativeScore(){
+        return wickets*40 + runs;
+    }
+
+    public int getMatches(){
+        return matches;
+    }
+}
+
+
+class WicketComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.getWicket() < p2.getWicket()){
+            return 1;
+        }
+        else if(p1.getWicket() > p2.getWicket()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+class RunComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.getRuns() < p2.getRuns()){
+            return 1;
+        }
+        else if(p1.getRuns() > p2.getRuns()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+
+class RunWicketComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.cumulativeScore() < p2.cumulativeScore()){
+            return 1;
+        }
+        else if(p1.cumulativeScore() > p2.cumulativeScore()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+
+
+
+class IPL{
+
+    // for putting all players into list of player objects.
+    public static List<Player> putPlayers(){
+
+        List<Player> listOfPlayers = new ArrayList<Player>();
+
+        try{
+
+            BufferedReader reader = new BufferedReader(new FileReader("./src/IPL_2021_data.csv"));
+
+            // for passing the header
+            reader.readLine();
+            String line = "";
+            while((line = reader.readLine()) != null){
+                String[] playerArray = line.split(",");
+                listOfPlayers.add(new Player(playerArray[0],playerArray[1],playerArray[2],Integer.valueOf(playerArray[3]),Integer.valueOf(playerArray[4]),Double.valueOf(playerArray[5]),Double.valueOf(playerArray[6]),Integer.valueOf(playerArray[7])));
+            }
+
+            reader.close();
+        }
+        catch(Exception e){
+            System.out.println("File not found "+ e);
+        }
+
+        return listOfPlayers;
+    }
+
+    // for fixing the match between team and stadium and storing in a csv file
+    public static void fixMatch(){
+        final int totalTeams = 8;
```

Comment:
Do not hardcode values like this. either pass the number of teams as arguments get it from the list of teams.